docs/manual/docs/cmd/yammer/report/report-groupactivityfilecounts.md

```diff
@@ -0,0 +1,42 @@
+# yammer report groupactivityfilecounts
+
+Gets the number of Yammer messages posted, read, and liked in groups
+
+## Usage
+
+```sh
+yammer report groupactivityfilecounts [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-p, --period <period>`|The length of time over which the report is aggregated. Supported values `D7,D30,D90,D180`
+`-f, --outputFile [outputFile]`|Path to the file where the report should be stored in
+`--query [query]`|JMESPath query string. See [http://jmespath.org/](http://jmespath.org/) for more information and examples
+`-o, --output [output]`|Output type. `text,json`. Default `text`
+`--pretty`|Prettifies `json` output
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+## Examples
+
+Gets the number of Yammer messages posted, read, and liked in groups for the last week
+
+```sh
+yammer report groupactivityfilecounts --period D7
+```
+
+Gets the number of Yammer messages posted, read, and liked in groups for the last week and exports the report data in the specified path in text format
+
+```sh
+yammer report groupactivityfilecounts --period D7 --output text --outputFile "groupactivityfilecounts.txt"
```

Comment:
Renamed from `groupactivityfilecounts` to `groupsactivitycounts` to match the issue description.